ledger/sandbox-on-x/src/classic/scala/com/daml/platform/ledger/ConfigConverter.scala

```diff
@@ -35,7 +35,6 @@ object ConfigConverter {
       // For Sandbox-on-X we don't offer that, so default to H2
       serverJdbcUrl = sandboxConfig.jdbcUrl.getOrElse(defaultH2SandboxJdbcUrl()),
       managementServiceTimeout = sandboxConfig.managementServiceTimeout,
```

Comment:
This conversion tagets a deprecated interface (`SandboxConfig`). We won't support or release new customer-facing products in the future based on this CLI, so no need to update it beyond its current state.